authors/apps/profiles/serializers.py

```diff
@@ -58,15 +61,42 @@ class Meta:
 
 
 class FollowListSerializer(serializers.ModelSerializer):
+	"""Serializes following/followers list."""
 	username = serializers.SlugRelatedField(read_only=True, slug_field='username')
 	follow_since = serializers.ReadOnlyField(source="created_time")
 
 	class Meta:
 		model = UserProfile
 		fields = ["username", "follow_since"]
 
+class NotificationSerializer(serializers.ModelSerializer):
+	"""Serializes notifications list."""
```

Comment:
Edit this comment to differentiate it from the `NotifySerializer`. You can add a comment why the notify serializer had to be different. i.e it's use case